### ISSUE TITLE
Preview render doesn't match page size

### ISSUE THREAD

**Issue by calculuschild** [stolksdorf/homebrewery#142]
**Browser Type/Version**: Google Chrome Version 51.0.2704.84 m

**Operating System**: Windows 7 64x

**Issue Description**: The preview on the right pane seems to have some code telling it to hide or display each page as it comes into view when scrolling up and down.  When using an irregular page size (e.g., landscape dimensions), this hide/render code seems to still be using the old dimensions; now that the page is much shorter vertically, the pages toward the bottom don't render until scrolled off the top of the screen.  Kind of hard to explain.  Maybe just look at my project and watch what happens when you scroll down the preview on the right.

**Brew code to reproduce**:

Here's my brew: http://homebrewery.naturalcrit.com/edit/HJxJgrVMN  I changed the dimensions as below:

```
<style>
.phb{
    width : 297mm;
    height : 230mm;
    padding : 6mm;
}
</style>
```

![image](https://cloud.githubusercontent.com/assets/5878534/16251462/75db673a-37e3-11e6-8637-5514af71187e.png)
Image not rendering

![image](https://cloud.githubusercontent.com/assets/5878534/16251475/8a4a82dc-37e3-11e6-8be6-007cb36b6792.png)
Ah!  There it finally rendered as I scrolled up farther.  Pages lower down get worse.

**Comment by calculuschild** [comment]
Hope this made sense.  It looks like something in your brewRenderer.jsx file where you determine viewablePageNumber.  My guess is its because you use

`Line 33 --- viewablePageNumber : Math.floor(e.target.scrollTop / PAGE_HEIGHT)`

which relies on a static PAGE_HEIGHT.  If the actual page height is different than this (as is my case) then the page that renders isn't necessarily the page in view.

**Comment by calculuschild** [comment]
This was fixed with 2.6.0.
